tests/fuzz/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,103 @@
+#![no_main]
+
+// TODO: HELP!
+// This contract sees the following build errors and I'm not sure what the right
+// way to fix them is:
+//
+// ❯ cargo +nightly fuzz build
+// Compiling test_fuzz v0.8.4 (/Users/leighmcculloch/Code/rs-soroban-sdk/tests/fuzz)
+// Compiling test_fuzz-fuzz v0.0.0 (/Users/leighmcculloch/Code/rs-soroban-sdk/tests/fuzz/fuzz)
+// error[E0433]: failed to resolve: could not find `size_hint` in `arbitrary`
+// --> fuzz_targets/fuzz_target_1.rs:18:10
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |          ^^^^^^^^^ could not find `size_hint` in `arbitrary`
+// |
+// = note: this error originates in the derive macro `Arbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)
+//
+// error[E0412]: cannot find type `Unstructured` in module `arbitrary`
+// --> fuzz_targets/fuzz_target_1.rs:18:10
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |          ^^^^^^^^^ not found in `arbitrary`
+// |
+// = note: this error originates in the derive macro `Arbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)
+//
+// error[E0412]: cannot find type `Result` in module `arbitrary`
+// --> fuzz_targets/fuzz_target_1.rs:18:10
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |          ^^^^^^^^^ not found in `arbitrary`
+// |
+// = note: this error originates in the derive macro `Arbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)
+// help: consider importing one of these items
+// |
+// 3  + use core::fmt::Result;
+// |
+// 3  + use core::result::Result;
+// |
+// 3  + use std::fmt::Result;
+// |
+// 3  + use std::io::Result;
+// |
+// and 2 other candidates
+// help: if you import `Result`, refer to it directly
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |
+//
+// error[E0433]: failed to resolve: could not find `Error` in `arbitrary`
+// --> fuzz_targets/fuzz_target_1.rs:18:10
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |          ^^^^^^^^^ could not find `Error` in `arbitrary`
+// |
+// = note: this error originates in the derive macro `Arbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)
+// help: consider importing one of these items
+// |
+// 3  + use core::error::Error;
+// |
+// 3  + use core::fmt::Error;
+// |
+// 3  + use soroban_sdk::Error;
+// |
+// 3  + use soroban_sdk::testutils::ed25519::Error;
+// |
+// and 4 other candidates
+// help: if you import `Error`, refer to it directly
+// |
+// 18 | #[derive(Arbitrary, Debug)]
+// |
```

Comment:
I need some help with this compile error ☝🏻

Reply:
I was able to resolve all my errors by importing `arbitrary = "1.3.0"` manually. So I think the re-export we have in the soroban-sdk isn't sufficient.

Reply:
The `Arbitrary` derive expects the `arbitrary` crate/module to be in scope. The `soroban_sdk::arbitrary` module reexports the `arbitrary` crate for this purpose, so

```
use soroban_sdk::arbitrary::arbitrary;
```

should fix this.